### PR TITLE
Update paiement.php

### LINKED ISSUE #14352: Bug: Non-numeric and header information error
# Instructions
*This is a template to help you report good issues. You may use [Github Markdown](https://help.github.com/articles/getting-started-with-writing-and-formatting-on-github/) syntax to format your issue report.*
*Please:*
- *replace the bracket enclosed texts with meaningful information*
- *remove any unused sub-section*

# Bug
[*Short description*]
Non-numeric and header information cannot be modify error when entering the customer payment in USD at the payment page.

## Environment
- **Version**: 12.0.1
- **OS**: Linux
- **Web server**: Apache
- **PHP**: cgi-fcgi 7.3.16
- **Database**: MySQL or MariaDB 5.6.40-84.0-log
- **URL(s)**: /compta/paiement.php?facid=50&action=create&accountid=2

## Expected and actual behavior
[*Verbose description*]

## Steps to reproduce the behavior
[*Verbose description*]

## [Attached files](https://help.github.com/articles/issue-attachments) (Screenshots, screencasts, dolibarr.log, debugging informations…)
[*Files*]
payment

### PR DESCRIPTION
**Pull request by asolslk**
fix #14352

### CODE REVIEW

File: htdocs/compta/paiement.php [hunk 1]
```diff
@@ -135,7 +135,7 @@
 			{
 				$cursorfacid = substr($key, 21);
 	            $multicurrency_amounts[$cursorfacid] = price2num(trim(GETPOST($key)));
-	            $multicurrency_totalpayment += $multicurrency_amounts[$cursorfacid];
+	            $multicurrency_totalpayment .= $multicurrency_amounts[$cursorfacid];
```
Comment (eldy) [inline htdocs/compta/paiement.php:138]: This looks stange.  The .= is used to concat string. For a total, we usually want a sum of numeric values, so  +=

Comment (asolslk) [reply htdocs/compta/paiement.php:138]: using += give the issue #14352, tested with .= and issue solved. not sure the real reason behind.

$multicurrency_totalpayment += $multicurrency_amounts[$cursorfacid];

Comment (eldy) [reply htdocs/compta/paiement.php:138]: The change remove the warning,but it returns a bad value of the total.

**Comment by eldy** [comment]
It seems this PR was set with "Discussion" or "PR to fix" tag, but no answer nor correction was provided since a long time. So request is automatically closed.
Please reopened and add a comment if you think this is an error.

**Pull request closed without merging**